GraphClient.go

```diff
@@ -304,6 +304,32 @@ func (g *GraphClient) CreateUser(userInput User, opts ...CreateQueryOption) (Use
 	return user, err
 }
 
+// SendMail sends a message.
+//
+// See https://docs.microsoft.com/en-us/graph/api/user-sendmail
+func (g *GraphClient) SendMail(userID string, message Message, saveToSentItems bool) error {
+	if g == nil {
```

Comment:
This is actually impossible here, because otherwise the function could not be called. However, if you move it to the `User` struct, it makes sense again if you check for `user.graphClient == nil`